src/index.html

```diff
@@ -203,35 +204,40 @@ <h2>HOBBIES & INTERESTS</h2>
       </ul>
 
       <!-- Skills -->
-      <section>
-        <h2>SKILLS</h2>
-        <ul>
-          <li>
-            <div>Microsoft Excel</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>Microsoft Word</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>Adobe</div>
-            <div>Proficient</div>
-          </li>
-          <li>
-            <div>Spanish</div>
-            <div>Advanced</div>
-          </li>
-          <li>
-            <div>PowerPoint</div>
-            <div>Proficient</div>
-          </li>
-          <li>
-            <div>French</div>
-            <div>Native</div>
-          </li>
-        </ul>
-      </section>
+      <div id="skills">
+        <section class="skills-section">
+          <h2>SKILLS</h2>
+          <hr id="skills-hr" />
```

Comment:
`hr` is not wrong, but I don't like the idea of achieving styles by including unnecessary html tag.

Remember good web developer use less code to achieve the same styles. Don't try to over complicated stuff.